package.json

```diff
@@ -4,22 +4,25 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "compile:theme": "babel src/theme --out-dir lib/theme --ignore *.test.js",
-    "compile:utils": "babel src/utils --out-dir lib/utils --ignore *.test.js",
-    "compile:constants": "babel src/constants --out-dir lib/constants --ignore *.test.js",
-    "compile:components": "babel src/components --out-dir lib/components --ignore *.test.js,example.js",
-    "compile:index": "babel src/index.js --out-file lib/index.js",
-    "install": "yarn compile:theme && yarn compile:utils && yarn compile:constants && yarn compile:components && yarn compile:index",
-    "test": "jest",
-    "start": "node bin/dev-server",
-    "debug": "node debug ./node_modules/.bin/jest -i",
-    "build": "RELEASE=true rm -rf dist && webpack --config webpack.config.js",
-    "lint-js": "eslint src docs",
-    "lint": "yarn lint-js",
     "ci": "yarn lint && yarn test",
-    "s3": "aws s3 sync ./dist/ s3://weaveworks-ui-components --acl public-read",
-    "copy:images": "cp -R src/images dist/images",
-    "release": "yarn lint && yarn test && yarn build && npm version patch && git push origin --follow-tags && yarn copy:images && yarn s3"
+    "lint": "eslint src docs",
+    "test": "jest",
+
+    "build": "yarn build:theme && yarn build:utils && yarn build:constants && yarn build:components && yarn build:index",
+    "build:theme": "babel src/theme --out-dir lib/theme --ignore *.test.js",
+    "build:utils": "babel src/utils --out-dir lib/utils --ignore *.test.js",
+    "build:constants": "babel src/constants --out-dir lib/constants --ignore *.test.js",
+    "build:components": "babel src/components --out-dir lib/components --ignore *.test.js,example.js",
+    "build:index": "babel src/index.js --out-file lib/index.js",
+
+    "release": "yarn ci && yarn build && npm version patch && git push origin --follow-tags",
+    "prepublish": "yarn ci && yarn build",
+
+    "docs:build": "RELEASE=true rm -rf dist && webpack --config webpack.config.js",
+    "docs:copy-images": "cp -R src/images dist/images",
+    "docs:s3": "aws s3 sync ./dist/ s3://weaveworks-ui-components --acl public-read",
+    "docs:upload": "yarn ci && yarn run docs:build && yarn docs:copy-images && yarn docs:s3",
```

Comment:
Not sure why e.g. `yarn docs:s3` works without `run` but then we have `yarn run docs:build`. Just to be on a safe side (and also for consistency), I think I'd do `yarn docs:*` -> `yarn run docs:*` and perhaps even `yarn ci` -> `yarn run ci`.

Reply:
Btw. why not include this in _relase_ or _publish_ flows? Is there a case when we'd want to e.g. release but not upload the docs immediately after?

Reply:
yarn doesn't require `run` to be used anymore. to me it feels more consistent to have `yarn test` `yarn build` `yarn randomscript` etc instead of some being prefixed with `run`. especially as scripts can all be overriden in package.json at the same level.